snmp/datadog_checks/snmp/data/default_profiles/fireeye.yaml

```
@@ -0,0 +1,183 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "fireeye"
+
+sysobjectid:
+  - 1.3.6.1.4.1.25597.1
+
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    table:
+      OID: 1.3.6.1.2.1.25.3.3
+      name: hrProcessorTable
+    symbols:
+      - OID: 1.3.6.1.2.1.25.3.3.1.2
+        name: cpu.usage
+    metric_tags:
+      - tag: hr_processor_frw_id
+        column:
+          OID: 1.3.6.1.2.1.25.3.3.1.1
+          name: hrProcessorFrwID
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.25.2.3.1.6.1 # hrStorageUsed.1 = Physical memory
+      name: memory.total
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.25.2.3.1.6.6 # hrStorageUsed.6 = Memory buffers
+      name: hrStorageUsed
+      tag: feMemoryBuffer
```

```suggestion
      tag: feMemoryBuffers
```

To use the same term used by the doc "Memory buffers"